x/feemarket/keeper/eip1559.go

```diff
@@ -59,7 +59,7 @@ func (k Keeper) CalculateBaseFee(ctx sdk.Context) *big.Int {
 	gasLimit := new(big.Int).SetUint64(math.MaxUint64)
 
 	// NOTE: a MaxGas equal to -1 means that block gas is unlimited
-	if consParams != nil && consParams.Block.MaxGas > -1 {
+	if consParams != nil && consParams.Block != nil && consParams.Block.MaxGas > -1 {
```

Comment:
actually, I think max-gas must not be omitted in the context of eip-1559, otherwise the base fee will keep decreasing.

Reply:
but if not exist we also can't use [default](https://github.com/evmos/ethermint/blob/main/app/utils.go#L46) which is unlimited

Reply:
Just raise an error?